### ISSUE TITLE
VictoryArea/VictoryLine do not render when animated

### ISSUE THREAD

**Issue by chrisbolin** [FormidableLabs/victory-native#73]
(reported on gitter by @parshap)

`<VictoryArea animate={{duration: 1000}}/>` does not render, but `<VictoryArea/>` renders fine.

**Comment by chrisbolin** [comment]
I confirmed that anytime `shouldAnimate` returns `true`, the component won't render anything.

**Comment by chrisbolin** [comment]
Traced this to the Clip Component. If I comment out this line in victory-native's `VictoryClipContainer` the Area charts render:
https://github.com/FormidableLabs/victory-native/blob/master/lib/components/victory-clip-container.js#L14

```jsx
export default class extends VictoryClipContainer {
  // Overrides method in victory-core
  renderClippedGroup(props, clipId) {
    const { style, events, children, className } = props;
    const nativeStyle = NativeHelpers.getStyle(style);
    const transform = NativeHelpers.getTransform(props.transform);
    const clipComponent = this.renderClipComponent(props, clipId);
    return (
      <G {...nativeStyle} {...events} {...transform} className={className}>
        {/* {clipComponent} THIS LINE */}
        <G clipPath={`url(#${clipId})`}>
          {children}
        </G>
      </G>
    );
  }
```

**Comment by chrisbolin** [comment]
This bug only effects components that use `VictoryClipContainer` for their `groupComponent`:
- VictoryArea
- VictoryLine

**Comment by chrisbolin** [comment]
A workaround for certain cases would be to override the `groupComponent` of the Area/Line chart with `react-native-svg`'s `G`:

```jsx
import { G } from "react-native-svg";

...
          <VictoryArea
            groupComponent={<G/>}
            ...
          />
```

Of course this will only work for simple charts, as more complex ones may overflow from the chart stage (hence the need for clipping)

**Comment by chrisbolin** [comment]
I believe I found the issue. If victory-native's `ClipPath` is ever given a `width` prop of `0`, it will not render ever, even if it changes later.

A hack that works is
```jsx
  renderClipPath(props, id) {
    props.width = props.width || 275;
```

**Comment by chrisbolin** [comment]
PR #74 fixes this issue.

**Comment by chrisbolin** [comment]
fixed in victory-native `v0.7.2`
